File: solutions/beecrowd/1805/1805.c
#include <stdint.h>
#include <stdio.h>

int main() {
    uint64_t first, last;

    while (scanf("%lld %lld", &first, &last) != EOF) {
        printf("%lld\n", (last - first + 1) * (first + last) / 2);
    }

    return 0;
}

File: solutions/beecrowd/1805/1805.go
package main

import "fmt"

func main() {
	var first, last uint64
	for {
		_, err := fmt.Scanf("%d %d", &first, &last)
		if err != nil {
			break
		}

		fmt.Println((last - first + 1) * (first + last) / 2)
	}
}
